who_is_active.sql

```diff
@@ -1824,6 +1824,7 @@ BEGIN;
 							WHEN
 								spy.wait_type LIKE N''PAGE%LATCH_%''
 								OR spy.wait_type = N''CXPACKET''
+                                OR spy.wait_type = N''CXCONSUMER''

```

Comment:
Formatting. Spaces, not tabs??! Awful!! Actually I think we should change the entire file to spaces but I'll do that in a separate PR.